src/CiviliansAccountsMicroservice/Civilians.Infrastructure/DbContext/Configuration/PassportsConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Civilians.Core.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace Civilians.Infrastructure.DbContext.Configuration
```

Comment:
You should iheret from IEntityTypeConfiguration. Then you can apply them all in onmodelcreating just in one line